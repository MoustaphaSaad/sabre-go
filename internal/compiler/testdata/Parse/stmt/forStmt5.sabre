for i, _ := range list {}