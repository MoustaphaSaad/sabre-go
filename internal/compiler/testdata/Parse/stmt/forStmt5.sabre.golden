(ForStmt
  (ForStmt-Range-Init
    (AssignStmt
      (IdentifierExpr IDENTIFIER(i))
      (IdentifierExpr IDENTIFIER(_))
      :=
      (UnaryExpr range
        (IdentifierExpr IDENTIFIER(list))
      )
    )
  )
  (ForStmt-Body
    (Block 0)
  )
)